package/libs/zlib/Makefile

```diff
@@ -46,27 +49,21 @@ define Package/zlib-dev/description
  This package includes the development support files.
 endef
 
-CONFIGURE_VARS := \
-	$(TARGET_CONFIGURE_OPTS) \
-	LDSHARED="$(TARGET_CC) -shared -Wl,-soname,libz.so.1" \
-	CFLAGS="$(TARGET_CFLAGS) $(FPIC)" \
-
-CONFIGURE_ARGS := \
-	--prefix=/usr \
-	--shared \
-	--uname=Linux \
-
-define Build/Compile
-	+$(MAKE) $(PKG_JOBS) -C $(PKG_BUILD_DIR) \
-		$(TARGET_CONFIGURE_OPTS) \
-		CFLAGS="$(TARGET_CFLAGS)" \
-		libz.a libz.so.$(PKG_VERSION)
-	mkdir -p $(PKG_INSTALL_DIR)
-	$(MAKE) -C $(PKG_BUILD_DIR) \
-		DESTDIR="$(PKG_INSTALL_DIR)" \
-		install
+define Package/zlib/config
+	source "$(SOURCE)/Config.in"
 endef
 
+TARGET_CFLAGS += $(FPIC)
```

Comment:
Why is this needed?

Reply:
It originates from https://github.com/lede-project/source/blob/master/package/libs/zlib/Makefile#L52
Turning to Google for guidance I found several occurrences about this:
https://bugzilla.redhat.com/show_bug.cgi?id=832545

Reply:
I see that other package Makefile's have it... so it's nothing out of the ordinary.

Was just wondering if it was related or a target-of-opportunity while you were cleaning things up.